tests/sentry/workflow_engine/endpoints/validators/test_base_data_condition_group.py

```diff
@@ -140,3 +125,64 @@ def test_create__with_conditions(self):
         assert condition.type == Condition.EQUAL
         assert condition.comparison == 1
         assert condition.condition_group == result
+
+
+class TestBaseDataConditionGroupValidatorUpdate(TestBaseDataConditionGroupValidator):
```

Comment:
We didn't have any update tests on this validator and I found a bug that I fixed where updating a data condition didn't actually work

Reply:
🙏 thanks for adding these.